webapp/src/package.ts

```diff
@@ -484,11 +484,16 @@ export class EditorPackage {
         return this.filterFiles(f => f.getName() == name)[0]
     }
 
```

Comment:
will want to count generated image files as auto generated above https://github.com/microsoft/pxt/blob/3611c3b7f06cf878561dc61fc38ec86344fb6851/webapp/src/package.ts#L144